modules/codemod/index.js

```diff
@@ -2,32 +2,32 @@
 'use strict';
 
 const {spawn} = require('child_process');
-require('colors');
+const chalk = require('chalk');
 
 const {_: commands, path} = require('yargs')
   .scriptName('canvas-kit-codemod')
-  .usage('$0 <transform> [path]'.brightBlue.bold)
-  .command('v5 [path]', 'Canvas Kit v4 > v5 migration transform'.gray, yargs => {
+  .usage(chalk.bold.blueBright('$0 <transform> [path]'))
```

Comment:
I prefer chalk's syntax. 😃

Reply:
Yes. I don't like when prototypes are modified. It is easy to make a mistake assuming the prototype is modified, but you didn't modify it. Simple changes can break in unpredictable ways.